apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.module.scss

```diff
@@ -0,0 +1,15 @@
+.preview,
```

Comment:
I notice also that we are moving from CSS to SCSS in this prototype. Just wanted to get that down for the record.